polyfill/lib/iscalendardate.mjs

```diff
@@ -0,0 +1,3 @@
+export function isCalendarDate(unit) {
```

Comment:
In this codebase we generally aren't using the one-export-per-file style; for now I'd suggest putting this into `lib/ecmascript.mjs`.

Reply:
Will do!  Also will try to update the spec text.  Probably going to get to this this weekend.